webui/src/main/java/org/polypheny/db/webui/Crud.java

```diff
@@ -1370,6 +1394,29 @@ Result getPlacements( final Request req, final Response res ) {
     }
 
 
+    /**
+     * Add or drop a data placement.
+     * Parameter of type models.Index: index name corresponds to storeUniqueName
+     * Index method: either 'ADD' or 'DROP'
+     */
+    Result addDropPlacement( final Request req, final Response res ) {
+        Index index = gson.fromJson( req.body(), Index.class );
```

Comment:
Wouldn't it be nicer to define a separate model for this? Alternatively: Generify the index model.